Dockerfile

```
@@ -0,0 +1,12 @@
+FROM python:3.7-alpine
+
+WORKDIR /px-ldap-replace
+
+COPY * ./
+
+RUN pip install -r requirements.txt && \
```

Always use long-form parameters!
```suggestion
RUN pip install --requirement=requirements.txt && \
```